app/static/js/scripts.js

```diff
@@ -42,25 +47,105 @@ function addOrganisations(response, map) {
     const xhttp = new XMLHttpRequest();
     xhttp.onreadystatechange = function () {
         if (this.readyState == 4 && this.status == 200) {
-            const organisations = JSON.parse(this.response);
-            var markers = L.markerClusterGroup();
-            for (let i = 0; i < organisations.length; i++) {
-                var marker = L.marker([organisations[i].latitude, organisations[i].longitude])
-                    .bindPopup(organisations[i].organisation_name + '<br>' + organisations[i].web_address)
-                    .openPopup();
-                markers.addLayer(marker);
-            }
-            map.addLayer(markers);
+            organisations = JSON.parse(this.response);
+            load_nace_codes();
+            markers = L.markerClusterGroup();
+            addMarkers(organisations)
         }
         else if (this.readyState == 4)
             alert(
-                `Unfortunately the list of organisations is currently unavailable. 
+                `Unfortunately the list of organisations is currently unavailable. 	
```

Comment:
Please remove the unneeded whitespace at the end of the line.

Reply:
Removed